.github/workflows/bazel.yml

```diff
@@ -291,9 +291,37 @@ jobs:
           run: |
             cd /workspaces/magma
             bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}"
-            bazel build lte/gateway/release:sctpd_deb_pkg lte/gateway/release:magma_deb_pkg \
+            bazel run //lte/gateway/release:release_build \
               --config=production \
               --profile=Bazel_build_package_profile
+            mkdir packages
+            mv /tmp/packages/*.deb packages
+      - name: Publish magma and sctpd
+        if: github.event_name == 'push' && github.repository_owner == 'magma'
+        env:
+          ARTIFACTORY_URL: "https://artifactory.magmacore.org/artifactory/debian-test/pool"
+          DEBIAN_META_INFO: ";deb.distribution=focal-ci;deb.component=main;deb.architecture=amd64"
+        run: |
+          cd packages
```

Comment:
Using [`working-directory:`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun) instead, might be cleaner.

Reply:
added